src/utils/L.Map.Sync.js

```diff
@@ -37,27 +35,29 @@ L.Map.include({
                 const newPos = getPosition(this._element)
                 const changePos = oldPos.subtract(newPos)
 
-                syncedMaps
-                    .filter(map => map !== origin)
-                    .forEach(map => {
+                syncedMaps.forEach(map => {
+                    if (map === origin) {
                         const mapPos = getPosition(
                             map.dragging._draggable._element
                         )
                         setPosition(
                             map.dragging._draggable._element,
                             mapPos.subtract(changePos)
                         )
-                    })
+                    }
+                })
             }
 
             this._onDragEndSync[id] = evt => {
-                const origin = evt.target
-                const center = origin.getCenter()
-                const zoom = origin.getZoom()
-
-                syncedMaps
-                    .filter(map => map !== origin)
-                    .forEach(map => setView.call(map, center, zoom, noAnimate))
+                const { target } = evt
+                const center = target.getCenter()
+                const zoom = target.getZoom()
+
+                syncedMaps.forEach(map => {
+                    if (map !== target) {
```

Comment:
Should be `map !== origin`

Reply:
I think target should work here, although origin would be the same.

Reply:
I've tested that target works as expected :-)

Reply:
Ah, didn't realize the event target was the map object.  Great.